apis/infix/atrium-api-infix/src/commonMain/kotlin/ch/tutteli/atrium/api/infix/en_GB/iteratorExpectations.kt

```diff
@@ -28,3 +26,29 @@ infix fun <T : Iterator<*>> Expect<T>.toHave(@Suppress("UNUSED_PARAMETER") next:
  */
 infix fun <T : Iterator<*>> Expect<T>.notToHave(@Suppress("UNUSED_PARAMETER") next: next): Expect<T> =
     _logicAppend { hasNotNext() }
+
+/**
+ * Expects that `this` expectation (an [Iterator]) has a next element and
+ * returns an [Expect] for the next element.
+ *
+ * @return The newly created [Expect] for the next element.
+ *
+ * @sample TODO
```

Comment:
just as reminder

Reply:
thx. added a link (file is changed though)